src/UNL/VisitorChat/Message/Record.php

```diff
@@ -86,4 +86,20 @@ function getConversation()
     {
         return \UNL\VisitorChat\Conversation\Record::getByID($this->conversations_id);
     }
+
+    /**
+     * Check if this message contains title 9 words
+     * 
+     * @return bool
+     */
+    public function hasTitle9Words()
+    {
+        foreach (\UNL\VisitorChat\Controller::$title9Words as $word) {
+            if (substr_count(strtolower($this->message), strtolower($word)) > 0) {
```

Comment:
Does the count matter or would `strpos` be faster? Not that it matter for our western needs, but we might want to consider starting to the the `mbstring` functions so we can support other langs.